assets/capability/capabilities/locked.json

```diff
@@ -104,7 +104,7 @@
           "no": "En lås er !{{låst|ulåst}}",
           "es": "Una cerradura está !{{cerrada|abierta}}",
           "da": "En lås er blevet !{{låst|låst op}}",
-          "ru": "Замок — !{{locked|unlocked}}",
+          "ru": "Замок !{{закрыт|открыт}}",
```

Comment:
It seems that `—` is removed, while it is not removed in `thermostat_mode.json`,